cluster/addons/rbac/node-bootstrap-binding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: node-bootstrap
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:node-bootstrapper
+subjects:
+- apiVersion: rbac/v1beta1
+  kind: Group
+  name: system:node-bootstrapper
```

Comment:
@liggitt the addon manager creates this and it will be enabled in GCE kube-up.sh deployments only. It will be a enabled in